src/main/java/core/basesyntax/figure/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.Color;
+import core.basesyntax.Figure;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, Color color) {
+        super.setColor(color);
```

Comment:
In child constructor we should call constructor of super class, so to fully resolve my previous comment you should create a constructor in Figure class, with Color color as argument. And call this constructor here like super(color)